addons/odoo_payments/models/adyen_account.py

```diff
@@ -98,6 +104,15 @@ def default_get(self, fields):
     transaction_payout_ids = fields.One2many('adyen.transaction.payout', 'adyen_account_id')
     payout_count = fields.Integer(compute='_compute_payout_count')
 
+    payment_journal_id = fields.Many2one(
+        string="Payment Journal", comodel_name='account.journal',
+        compute='_compute_payment_journal_id', inverse='_inverse_payment_journal_id',
+        help="The journal in which the successful transactions are posted",
+        domain="[('type', '=', 'bank'), ('company_id', '=', company_id)]")
+
+    # UX flag to know if the user has to select/create a journal or if it will be created automatically for him.
+    need_to_provide_payment_journal = fields.Boolean(default=_default_need_to_provide_payment_journal, readonly=True)
```

Comment:
Should be a computed UX field, not a stored boolean (dummy UX fields are not worth storing in db)

Reply:
Yes, I agree, done :+1: